tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/TraceAnnotationsTests.cs

```diff
@@ -103,7 +103,7 @@ public async Task SubmitTraces()
             {
                 var spans = agent.WaitForSpans(expectedSpanCount);
 
-                var orderedSpans = spans.OrderBy(s => s.Start);
+                var orderedSpans = spans.OrderBy(s => s.Start).ToList();
```

Comment:
nit: By removing the ToList() you'll now get a resharper warning about multiple iterations, so I'll no doubt add the ToList() back at some point 😉

Reply:
I think I _added_ `ToList()` (because I got the R# warning), not removed it.

Reply:
Oh yeah, my bad 😂 🤦‍♂️